docs/quickstart.md

```diff
@@ -0,0 +1,176 @@
+# Initial Setup / Operations
+
+`SSO` is used to provide single-sign-on authentication and authorization for internal web
+applications, however, there are a few one-time prerequisites that need to be handled manually.
```

Comment:
This sentence is a bit odd, the first clause and second clause seem like the should be related because of the "however," but I think they are totally unrelated.  By the time a person has reached this doc, I think they know what sso is.

I think we can just drop it altogether, as it doesn't provide much context that we're not already providing immediately below at the beginning of the "Google OAuth Setup" step!